app/src/main/java/cz/muni/fi/pv256/movio2/uco_410371/MainFragment.java

```diff
@@ -0,0 +1,177 @@
+package cz.muni.fi.pv256.movio2.uco_410371;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.List;
+
+import cz.muni.fi.pv256.movio2.uco_410371.adapters.EmptyRecyclerViewAdapter;
+import cz.muni.fi.pv256.movio2.uco_410371.adapters.MovieRecyclerViewAdapter;
+import cz.muni.fi.pv256.movio2.uco_410371.dummy.DummyContent;
+
+/**
+ * Main Fragment
+ * Created by Benjamin Varga on 6.10.2016.
+ */
+
+public class MainFragment extends Fragment implements View.OnClickListener {
+
+    private static final String TAG = MainFragment.class.getName();
+
+    private boolean mTwoPane;
+
+    public MainFragment() {}
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        Log.d(TAG, "onAttach: ");
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Log.d(TAG, "onCreate: ");
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        Log.d(TAG, "onCreateView: ");
+        View view = inflater.inflate(R.layout.fragment_main, container, false);
+
+        if (view.findViewById(R.id.movie_detail_container) != null) {
+            mTwoPane = true;
+        }
+
+        RecyclerView recyclerView = (RecyclerView)view.findViewById(R.id.recyclerView_main);
+        recyclerView.setHasFixedSize(true);
+        recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
+
+        List<Object> list = DummyContent.ITEMS;
+
+        if (list == null || list.size() == 0) {
+            EmptyRecyclerViewAdapter emptyRecyclerViewAdapter = new EmptyRecyclerViewAdapter("NO DATA");
```

Comment:
Netreba vymienat dva recycler adaptery. Malo by stacit nastavit emptyview do RecyclerView a to sa zobrazi ak nebudu ziadne polozky v zozname.